tools/extensions/generate_extension_db.py

```diff
@@ -103,4 +114,5 @@ def GetExtensionMetadata(target):
   extension_db['envoy.upstreams.http.http_protocol_options'] = GetExtensionMetadata(
       '//source/extensions/upstreams/http:config')
```

Comment:
afaict it was these refs to `//source/...` that fail if i dont include the ``envoy_all_extensions()`` in the `BUILD`

im wondering if its just because that brings the source into the build rather than because ``envoy_all_extensions`` is required specifiically

yep, re the visibility - i just wasnt sure how to make it work without - ill try again...